openedx/edx-ora2
# chore: update help text of top responses setting and translations

**TL;DR -** Add a small change on help text of "Top Responses" setting to make it clearer to instructor. This is a re-done PR  in place of #1870 as @ngdangdat couldn't get the CLA Signed. 

JIRA: NA

**What changed?**

- Help text of the "Top Responses" setting
- Related translation files

**Developer Checklist**


**Testing Instructions**

* Checkout branch [open-craft:update-help-text-of-top-reponses-setting](https://github.com/open-craft/edx-ora2/tree/update-help-text-of-top-reponses-setting)
 * Install `edx-ora2` xblock on the feature branch to Studio (CMS)
 * Restart CMS
 * Add a ORA block in a unit
 * Edit > Settings
 * Make sure the changed text show up properly

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @openedx/content-aurora
